libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -1237,4 +1237,34 @@ int lua_GCS_command_int(lua_State *L)
 }
 #endif
 
+#if HAL_ENABLE_DRONECAN_DRIVERS
+/*
+  get FlexDebug from a DroneCAN node
+ */
+int lua_DroneCAN_get_FlexDebug(lua_State *L)
+{
+    binding_argcheck(L, 4);
+
+    const uint8_t bus = get_uint8_t(L, 1);
+    const uint8_t node_id = get_uint8_t(L, 2);
+    const uint16_t msg_id = get_uint16_t(L, 3);
+    uint32_t tstamp_us = get_uint32(L, 4, 0, UINT32_MAX);
```

Comment:
```suggestion
    uint32_t tstamp_us = coerce_to_uint32_t(L, 4);
```
Don't need the range check.